### PR TITLE
Update protocol.json for NEO3

### LINKED ISSUE #454: Create a new magic for NEO3 testnet in protocol.json
Create a new magic for NEO3 testnet in protocol.json

### PR DESCRIPTION
**Pull request by erikzhang**
Closes #454

### CODE REVIEW

File: neo-cli/protocol.json [hunk 1]
```diff
@@ -13,16 +13,6 @@
...
-      "seed2.ngd.network:10333",
-      "seed3.ngd.network:10333",
-      "seed4.ngd.network:10333",
-      "seed5.ngd.network:10333",
-      "seed6.ngd.network:10333",
-      "seed7.ngd.network:10333",
-      "seed8.ngd.network:10333",
-      "seed9.ngd.network:10333",
-      "seed10.ngd.network:10333",
       "seed1.neo.org:10333",
```
Comment (igormcoelho) [inline neo-cli/protocol.json]: Erik, shouldn't we default it to TestNet by now?

      "seed1t.neo.org:20333",
      "seed2t.neo.org:20333",
      "seed3t.neo.org:20333",
      "seed4t.neo.org:20333",
      "seed5t.neo.org:20333"

**Comment by shargon** [comment]
Are active now?

**Comment by lock9** [comment]
Hi @erikzhang, 
This is the configuration that is currently working for testnet.

```json
{
  "ProtocolConfiguration": {
    "Magic": 161616,
    "AddressVersion": 23,
    "MillisecondsPerBlock": 15000,
    "StandbyValidators": [
      "0214baf0ceea3a66f17e7e1e839ea25fd8bed6cd82e6bb6e68250189065f44ff01",
      "023e9b32ea89b94d066e649b124fd50e396ee91369e8e2a6ae1b11c170d022256d",
      "02a7834be9b32e2981d157cb5bbd3acb42cfd11ea5c3b10224d7a44e98c5910f1b",
      "02ba2c70f5996f357a43198705859fae2cfea13e1172962800772b3d588a9d4abd",
      "03009b7540e10f2562e5fd8fac9eaec25166a58b26e412348ff5a86927bfac22a2",
      "030205e9cefaea5a1dfc580af20c8d5aa2468bb0148f1a5e4605fc622c80e604ba",
      "03408dcd416396f64783ac587ea1e1593c57d9fea880c8a6a1920e92a259477806"
    ],
    "SeedList": [
      "seed1t.neo.org:20333",
      "seed2t.neo.org:20333",
      "seed3t.neo.org:20333",
      "seed4t.neo.org:20333",
      "seed5t.neo.org:20333"
    ]
  }
}
```

**Comment by erikzhang** [comment]
@lock9 The TestNet has not yet started.

**Pull request merged**